packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/serialization_descriptors.py

```diff
@@ -0,0 +1,177 @@
+"""Serialization Descriptors."""
+
+from json import loads
+from typing import Any, List
+
+from google.protobuf import descriptor_pb2, descriptor_pool
+from google.protobuf.descriptor_pb2 import FieldDescriptorProto
+
+from ni_measurement_plugin_sdk_service._annotations import ENUM_VALUES_KEY
+from ni_measurement_plugin_sdk_service._internal.parameter.metadata import (
+    ParameterMetadata,
+)
+from tests.utilities.stubs.loopback.types_pb2 import ProtobufColor
+
+
+def _get_output_enum_type(
+    metadata_enum_list: List[str],
+    file_descriptor: descriptor_pb2.FileDescriptorProto,
+) -> Any:
+    """Get's matching enum class from 'file_descriptor'.
+
+    Args:
+        metadata_enum_list (List[str]): Enum names from metadata.annotations.
+
+        file_descriptor: Descriptor of proto file.
+
+    Returns:
+        Any: Matching enum class in a str type or None when enum is protobuf.
+    """
+    for enum_type in file_descriptor.enum_type:
+        enum_names = [enum_value.name for enum_value in enum_type.value]
+        if sorted(metadata_enum_list) == sorted(enum_names):
+            return enum_type.name
+    return None
```

Comment:
Why is this an Any? From the code, it looks like an `Optional[str]`, assuming that `enum_type.name` is a`str` (is it?)

Reply:
It is, I didn't know how to make the return type be 2 different types so that's why I made it as a Any.

Reply:
Whenever you're dealing with two different possible types, you use `typing.Union[type1, type2]` (parameter or return). `typing.Optional[foo]` is shorthand for `typing.Uniont[None, foo]`.